public/javascripts/SVLabel/spec/SVLabel/modal/ModalMissionCompleteMap-spec.js

```diff
@@ -0,0 +1,317 @@
+describe("ModalMissionCompleteMap", function () {
+    var $uiModalMissionCompleteFixture;
+    var neighborhood;
+    var mission;
+    var map;
+
+    // Mocks
+    function MissionMock () {
+        this.properties = {
+            coverage: null,
+            label: null,
+            distance: null,
+            distanceFt: null,
+            distanceMi: null
+        };
+    }
+    MissionMock.prototype.getProperty = function (key) {
+        return this.properties[key];
+    };
+
+    function NeighborhoodMock() {
+        this.properties = {
+            name: null,
+            regionId: null,
+            center: null,
+            geoJSON: null
+        };
+    }
+    NeighborhoodMock.prototype.getProperty = function (key) {
+        return this.properties[key];
+    };
+    NeighborhoodMock.prototype.center = function () {
+        return this.properties.center;
+    };
+    NeighborhoodMock.prototype.getGeoJSON = function () {
+        return this.properties.geoJSON;
+    };
+
+    function TaskContainerMock(){
+        this.properties = {};
+    }
+
+    function TaskMock() {
+        this.properties = {
+            streetEdgeId: null,
+            feature: null
+        };
+    } 
+
+    TaskMock.prototype.getStreetEdgeId = function () {
+        return this.properties.streetEdgeId;
+    };
+
+    TaskMock.prototype.getFeature = function () {
+        return this.properties.feature;
+    };
+
+    TaskMock.prototype.getGeoJSON = function () {
+        var geoJSON = {
+            "type":"FeatureCollection",
+            "features": [this.properties.feature]
+        };
+        return geoJSON;
+    };
+
+    beforeEach(function () {
+        $uiModalMissionCompleteFixture = $('<div id="modal-mission-complete-map"></div>');
+
+        // leaflet relies on a map container in the html
+        $('body').append($uiModalMissionCompleteFixture);
```

Comment:
This is a little problematic; (i) this relies on the presence of HTML file and (ii) it modifies the state of the spec runner file.  For (i), I suppose this will not pass if you run `npm test` from command line... (ii) It's ok, but if you are going to change the state, remove what you have appended in `afterEach` method.

Are you sure you cannot simply make a fixture for a map? (i.e., `$('<div id="map" />` or something). If you can, that would be better as the test will be more independent.